homeassistant/components/enphase_envoy/coordinator.py

```diff
@@ -50,8 +53,10 @@ def __init__(
         self.password = entry_data[CONF_PASSWORD]
         self._setup_complete = False
         self.envoy_firmware = ""
+        self.interface = None
```

Comment:
```suggestion
        self.interface: EnvoyInterfaceInformation | None = None
```

Reply:
nevermind you have it defined in https://github.com/home-assistant/core/pull/143758/files#diff-3b7ccffbd9fe52a66d6504139e4a084e40e7c4961761cd2a10908c3c1dc5744aR44